app/src/main/java/com/kickstarter/services/KSApolloClient.kt

```diff
@@ -1169,11 +1158,13 @@ private fun projectTransformer(projectFragment: FullProject?): Project {
         .staticUsdRate(staticUSDRate)
         .usdExchangeRate(usdExchangeRate)
         .updatedAt(updatedAt)
-        // .unreadMessagesCount() TODO: unread messages can be fetched form the User Object
-        // .unseenActivityCount() TODO: unseen activity can be fetched form the User Object
+        // .unreadMessagesCount() unread messages can be fetched form the User Object
```

Comment:
Nit: Do we need this commented out code? If not we should remove it, unless it's providing context.

Reply:
It's for providing context, as those fields have value when fetched from V1, but  on GraphQL they do not exist under Project